openshift/templates/agent/agent-deploy.bc.param

```diff
@@ -12,14 +12,14 @@ NAMESPACE=8ad0ea
 SUFFIX=-bc
 APP_NAME=orgbook
 APP_GROUP=aries-vcr
-APPLICATION_DOMAIN=orgbook-agent-dev.apps.silver.devops.gov.bc.ca 
-APPLICATION_ADMIN_DOMAIN=orgbook-agent-admin-dev.apps.silver.devops.gov.bc.ca
+APPLICATION_DOMAIN=agent-dev.orgbook.gov.bc.ca
+APPLICATION_ADMIN_DOMAIN=agent-admin-dev.orgbook.gov.bc.ca
 ROUTE_TIMEOUT=120s
 IMAGE_NAMESPACE=8ad0ea-tools
 NAMESPACE_NAME=8ad0ea
 TAG_NAME=dev
 AGENT_LABEL_NAME=OrgBook BC Agent
-AGENT_BASE_URL=https://orgbook-agent-dev.apps.silver.devops.gov.bc.ca
+AGENT_BASE_URL=https://agent-dev.orgbook.gov.bc.ca
```

Comment:
While the parameters in this base config are overridder per-profile, we may want to have them match the `dev` environment for clarity.

Reply:
updated